src/accommodation-agent/Models/Accommodation.cs

```diff
@@ -0,0 +1,34 @@
+namespace AccommodationAgent.Models;
+
+public class Accommodation
+{
+    public required string Id { get; set; }
+    public required string Name { get; set; }
+    public required string Type { get; set; }
```

Comment:
the accomodation type should be an enum, not a string. For the enum values use the most common accomodation types you can think of.